src/universal/common/components/AppFooter/AppFooter.tsx

```diff
@@ -1,56 +1,65 @@
 import * as React from 'react';
 import {Link} from 'react-router';
+import {FormattedLink, FormattedMessage} from 'universal/common/components';
 
 export default class AppFooter extends React.Component<any, any> {
-    public render() {
-      const css = require('./AppFooter.scss');
+  public render() {
+    const css = require('./AppFooter.scss');
 
-      return (
-        <div className={css.AppFooter}>
-        <div className={css.AppFooter_sectionWrapper}>
+    return (
+      <div className={css.AppFooter}>
+      <div className={css.AppFooter_sectionWrapper}>
 
-          <div className={css.AppFooter_section}>
-            <div className={css.AppFooter_logo}>
-              <a href="/">EVER<b>REAL</b></a>
-            </div>
-            <div className={css.AppFooter_copyright}>© EVERREAL {new Date().getFullYear()}</div>
-
-            <ul className={css.AppFooter_sectionLinks}>
-              <li>
-                <a href="#"><img src="https://s3.eu-central-1.amazonaws.com/s3-production/assets/images/common/flags/de.png" alt="de" title="Deutsch" />&nbsp;Deutsch</a>
-              </li>
-              <li>
-                <a href="#"><img src="https://s3.eu-central-1.amazonaws.com/s3-production/assets/images/common/flags/en.png" alt="en" title="English" />&nbsp;English</a>
-              </li>
-            </ul>
+        <div className={css.AppFooter_section}>
+          <div className={css.AppFooter_logo}>
+            <a href="https://www.everreal.de">EVER<b>REAL</b></a>
           </div>
+          <div className={css.AppFooter_copyright}>© EVERREAL {new Date().getFullYear()}</div>
 
-          <div className={css.AppFooter_section}>
-            <div className={css.AppFooter_sectionTitle}>
-              Company
-            </div>
-            <ul className={css.AppFooter_sectionLinks}>
-              <li><a href="#">About us</a></li>
-              <li><a href="#">Imprint</a></li>
-              <li><a href="#">Privacy policy</a></li>
-            </ul>
-          </div>
+          <ul className={css.AppFooter_sectionLinks}>
+            <li>
+              <a href="/de-DE/login" title="Deutsch">
```

Comment:
I am thinking that maybe something like `location.href.replace(currentLanguage, 'de-DE')` could do the trick also, but the current language should be read from `route.params.language`, not from `this.context.intl.locale`, because an url like `/whatever/login/sub-route` will have `route.params.language === 'whatever'` and `this.context.intl.locale === defaultLanguage === 'en-US'`.